### PR TITLE
Update production-debian.md with let's encrypt

### PR DESCRIPTION
**Pull request by iorseth**
Hi, I've test your documentation for my server and for the ssl, I've used the free and open source certificate authority : let's encrypt. 
It works very good, Please test and enjoy :).

### CODE REVIEW

**Comment by coreyhulen** [comment]
Thanks @iorseth for the pull request! 

Per the [**CONTRIBUTING.md**](https://github.com/mattermost/platform/blob/master/CONTRIBUTING.md#submitting-a-pull-request) file displayed when you created this pull request, we need to add you to the list of [approved contributors](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true) for the Mattermost project.

**Please help complete the Mattermost [contribution license agreement](http://www.mattermost.org/mattermost-contributor-agreement/)?**

This is a standard procedure for many open source projects. Your form should be processed within 24 hours and reviewers for your pull request will be able to proceed. 

Please let us know if you have any questions. 

We are very happy to have you join our growing community!

File: doc/install/Production-Debian.md [hunk 1]
```diff
@@ -264,7 +264,16 @@ exit 0
     Common Name (e.g. server FQDN or YOUR name) []:mattermost.example.com
     Email Address []:admin@mattermost.example.com
 ```
-1. Modify the file at `/etc/nginx/sites-available/mattermost` and add the following lines
+1. You can use a free and an open certificate security like let's encrypt, this is how to proceed
+  * ```apt-get install git```
+  * ```git clone https://github.com/letsencrypt/letsencrypt```
+  * ```cd letsencrypt```
+  1. Be sure that the port 80 is not use
```
Comment (jwilander) [inline doc/install/Production-Debian.md]: Should be "in use" correct?

**Comment by coreyhulen** [comment]
I think we should remove `2. For simplicity we will generate a test certificate.` and also update all the docs at Production-*.md.  They all have the same section.

**Comment by jwilander** [comment]
Hi @iorseth, just wondering if you're able to update the PR based on @coreyhulen's comment? If not, I'll close this PR and we'll use it as the base for a new PR to update the documentation.

**Comment by coreyhulen** [comment]
Docs have been moved to https://github.com/mattermost/docs and I incorporated this change.

**Pull request closed without merging**